test/wallet.spec.ts

```diff
@@ -601,5 +601,64 @@ describe('Wallet', () => {
         assert.strictEqual(tx.outs.length, 3);
       });
     });
+
+    describe('transfer uncolored tpc', () => {
+      it('should build transaction', async () => {
+        const { wallet: alice, dataStore } = createWallet('prod');
+        const stub = setUpStub(alice);
+        stub.onFirstCall().returns(new Promise(resolve => resolve(response)));
+        await alice.importWif(wif);
+        await dataStore.add(utxos);
+        const tx = await alice.transfer(
+          [
+            {
+              colorId: BaseWallet.COLOR_ID_FOR_TPC,
+              amount: amount,
+              toAddress: toAddress,
+            },
+          ],
+          changePubkeyScript,
+        );
+        assert.strictEqual(tx.ins.length, 1);
+        assert.strictEqual(tx.outs.length, 2);
+      });
```

Comment:
`tx.out[0].script` has color id and OP_COLOR like `000000000000000000000000000000000000000000000000000000000000000000 OP_COLOR` ahead of a p2phk script.
This should not be here.

Reply:
Fixed at f9c4aa7